packages/container-registry-v5/commands/release.js

```diff
@@ -93,17 +95,23 @@ let release = async function (context, heroku) {
     return
   }
 
-  if (release.status === 'failed') {
+  let releaseStatus = release.status
+
+  if (releaseStatus === 'failed') {
     cli.exit(1, 'Error: release command failed')
-  } else if ((release.status === 'pending') && release.output_stream_url) {
+  } else if ((releaseStatus === 'pending') && release.output_stream_url) {
     cli.log('Running release command...')
     await streamer(release.output_stream_url, process.stdout)
 
-    let finishedRelease = await heroku.request({
-      path: `/apps/${context.app}/releases/${release.id}`
-    })
+    while (releaseStatus === 'pending') {
```

Comment:
@heroku/build - Is this ever likely to not return and we need to add a timeout?

Reply:
We have seen a couple eventbus incidents where events were missed and releases remained in pending forevers.
During those eventbus incidents, events could be very delayed too. So yes, adding a timeout seems like a good thing to do.

Reply:
Got it!